Library/Homebrew/env_config.rb

```diff
@@ -333,7 +333,7 @@ module EnvConfig
         boolean:     true,
       },
       HOMEBREW_PIP_INDEX_URL:                    {
-        description:  "If set, `brew install <formula>` will use this URL to download PyPI package resources.",
+        description:  "If set, `brew install` <formula> will use this URL to download PyPI package resources.",
```

Comment:
I think I prefer the old way more since it groups together the command with the arguments into one logical unit.

Reply:
It's a quirk of the fact that the docs need to be processed by both a Markdown processor for web pages and [ronn](https://github.com/rtomayko/ronn) for manpages. This format is required because the original results in `brew install *formula*` on the webpage, rather than `brew install` *`formula`* (with "formula" in italics to indicate it's a placeholder word within a suggested command).